packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -14,11 +14,11 @@
 Summary: Universal command-line interface for Foreman
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.15
```

Comment:
Mind bumping this to 0.16 while you're at it to match https://github.com/theforeman/hammer-cli/commit/298d6077c24a340e0c135c3e8af3097e82ccfc8a